packages/stack/storage/package.json

```diff
@@ -50,12 +51,13 @@
     "core-js": "3.4.3",
     "embark-core": "^5.3.0-nightly.1",
     "embark-i18n": "^5.2.3",
-    "embark-utils": "^5.3.0-nightly.0",
-    "lodash.clonedeep": "4.5.0",
-    "shelljs": "0.8.3"
+    "embark-utils": "^5.3.0-nightly.0"
   },
   "devDependencies": {
+    "@babel/core": "7.8.3",
+    "babel-jest": "25.1.0",
     "embark-solo": "^5.2.3",
+    "embark-utils": "^5.3.0-nightly.0",
```

Comment:
You put embark-utils both in dependencies and dev-deps, you can remove the latter then.